Ollivanders/src/net/pottercraft/ollivanders2/spell/APPARATE.java

```diff
@@ -295,6 +295,13 @@ private boolean canApparateTo(@NotNull Location destination)
             return false;
         }
 
+        // check if x or z coordinates exceed max set in configuration
+        if ((destination.getX() >= p.getConfig().getInt("maxApparateDistance")) || (destination.getZ() >= p.getConfig().getInt("maxApparateDistance")))
+        {
+            common.printDebugMessage("Player is unable to teleport that far", null, null, false);
```

Comment:
We probably also want to tell the player that they cannot go that far rather than have the command just die.

Reply:
Fixed in latest commit!